docs/source/sctool/partials/sctool_restore.yaml

```diff
@@ -0,0 +1,158 @@
+
+    '*' - matches any number of any characters including none
+    '?' - matches any single character
+    '[abc]' - matches one character given in the bracket
+    '[a-z]' - matches one character from the range given in the bracket
+
+    Patterns are evaluated from left to right.
+    If a pattern starts with '!' it unselects items that were selected by previous patterns i.e. 'a?,!aa' selects *ab* but not *aa*.
+
+- name: location
+  shorthand: L
+  default_value: '[]'
+  usage: |
+    A list of backup locations separated by a comma, specifies places where restored backup is stored, the format is `[<dc>:]<provider>:<bucket>`.
+    The '<dc>' parameter is optional it allows to specif datacenter which nodes will be used to restore data
+    from this location in a multi-dc setting, it must match Scylla nodes datacenter.
+    By default, all live nodes are used to restore data from specified locations.
+    Note that specifying datacenters closest to backup locations might reduce download time of restored data.
+    The supported storage '<provider>'s are 'azure', 'gcs', 's3'.
+    The 'name' parameter is a bucket name, it must be an alphanumeric string and **may contain a dash and or a dot, but other characters are forbidden**.
+- name: name
+  usage: |
+    Task name that can be used insead of ID.
+- name: num-retries
+  shorthand: r
+  default_value: "3"
+  usage: |
+    Number of times a task reruns following a failure.
+- name: parallel
+  default_value: "1"
+  usage: |
+    The maximum number of Scylla restore jobs that can be run at the same time (on different SSTables).
+    Each node can take part in at most one restore at any given moment.
+- name: restore-schema
+  default_value: "false"
+  usage: |
+    Specifies restore type (alternative to '--restore-tables' flag).
+    Restore will recreate schema by targeting 'system_schema.*' tables only ('--keyspace' flag mustn't be set).
+    It requires that restored keyspaces aren't present in the cluster.
+    For the schema to be visible, restart of the entire cluster is required after completion.
+    See Scylla cluster restart docs: https://docs.scylladb.com/stable/operating-scylla/procedures/config-change/rolling-restart.html for more details.
+- name: restore-tables
+  default_value: "false"
+  usage: "Specifies restore type (alternative to '--restore-schema' flag).\nRestore will recreate contents of tables specified by '--keyspace' flag.\nIt requires that correct schema of restored tables is already present in the cluster (schema can be restored using '--restore-schema' flag).\nMoreover, in order to prevent situation in which current tables' contents overlaps restored data,\ntables should be truncated before initializing restore. \nFor the table contents to be visible, repair of the entire cluster is required after completion.\nSee Scylla Manager repair docs: https://manager.docs.scylladb.com/stable/sctool/repair.html#sctool-repair for more details.\n"
```

Comment:
```suggestion
  usage: "Specifies restore type (alternative to '--restore-schema' flag).\nRestore will recreate contents of tables specified by '--keyspace' flag.\nIt requires that correct schema of restored tables is already present in the cluster (schema can be restored using '--restore-schema' flag).\nMoreover, in order to prevent situation in which current tables' contents overlaps restored data,\ntables should be truncated before initializing restore. \nFor the table contents to be visible, repair of the entire cluster is required after completion.\nSee ref:`sctool restore <sctool-restore>` for details.\n"
```

Reply:
The problem is that `sctool command` descriptions (files in `partials/*`) in documentation are generated from the `sctool command --help` output.
In this case, suggested change results in:
```
--restore-tables sctool restore <sctool-restore>          Specifies restore type (alternative to '--restore-schema' flag).
                                                          Restore will recreate contents of tables specified by '--keyspace' flag.
                                                          It requires that correct schema of restored tables is already present in the cluster (schema can be restored using '--restore-schema' flag).
                                                          Moreover, in order to prevent situation in which current tables' contents overlaps restored data,
                                                          tables should be truncated before initializing restore.
                                                          For the table contents to be visible, repair of the entire cluster is required after completion.
                                                          See ref:sctool restore <sctool-restore> for details.

```
Because of that, IMHO it's better to leave full links in flag descriptions instead of changing them to refs which aren't properly rendered in terminal.

I would like to avoid situations where after each generation of `partials/*` we would have to remember to change some links into refs, because it would eventually be forgotten. The best solution would be to automate the process of such substitution, but it does not look that easy to me.